pkg/discovery/step.go

```diff
@@ -6,7 +6,7 @@ import (
 	"slices"
 )
 
-var StepFileName = regexp.MustCompile(`^(\d\d)-(.*)\.(?:yaml|yml)$`)
+var StepFileName = regexp.MustCompile(`^(\d\d+)-(.*)\.(?:yaml|yml)$`)
```

Comment:
```suggestion
var StepFileName = regexp.MustCompile(`^(d+)-(.*)\.(?:yaml|yml)$`)
```

Seems safer to also allow 1-digit prefixes like [kuttl does](https://github.com/kudobuilder/kuttl/blob/c7777a0bf2b9398b6fd294cbccaa4be381c5f3ab/pkg/test/step.go#L33).

Reply:
I started with digit prefix but was too lazy to look at the kuttle code 🙈 
Thanks, i will update

Reply:
Fixed